content/en/docs/concepts/scheduling-eviction/pod-eviction.md

```diff
@@ -0,0 +1,397 @@
+all the containers and they are equal. These pods will never be evicted because
+of another pod's resource consumption. If a system daemon (such as `kubelet`,
+`docker`, and `journald`) is consuming more resources than were reserved via 
+`system-reserved` or `kube-reserved` allocations, and the node only has
+`Guaranteed` or `Burstable` pods using less resources than requests left on it,
+then the kubelet must choose to evict one of these pods to preserve node stability
+and to limit the impact of resource starvation on other Pods. In this case, it
+will choose to evict pods of lowest Priority first.
+
+When the kubelet evicts pods to reclaim disk resources, it sorts pods by QoS and
+terminates pods using the largest amount of disk. When the kubelet responds to
+`inode` starvation, it uses the QoS to make eviction decisions. The kubelet sorts
+pods differently based on whether the node has a dedicated `imagefs` filesystem.
+
+### With `imagefs`
+
+If `nodefs` is triggering evictions, the kubelet sorts pods based on `nodefs`
+usage (`local volumes + logs of all containers`).
+
+If `imagefs` is triggering evictions, the kubelet sorts pods based on the
+writable layer usage of all containers.
+
+### Without `imagefs`
+
+If `nodefs` is triggering evictions, the kubelet sorts pods based on their total
+disk usage (`local volumes + logs & writable layer of all containers`)
+
+## Minimum eviction reclaim
+
+In some cases, pod eviction only reclaims a small amount of the starved resource.
+This can lead to the kubelet repeatedly hitting the configured eviction thresholds
+and triggering multiple evictions. 
+
+You can use the `eviction-minimum-reclaim` flag to configure a minimum reclaim
+amount for each resource. When the kubelet notices that a resource is starved,
+it continues to reclaim that resource until it reclaims the quantity you specify. 
+
+For example, the following configuration sets minimum reclaim amounts: 
+
+```
+--eviction-hard=memory.available<500Mi,nodefs.available<1Gi,imagefs.available<100Gi
+--eviction-minimum-reclaim="memory.available=0Mi,nodefs.available=500Mi,imagefs.available=2Gi"`
+```
+
+In this example, if the `nodefs.available` signal meets the eviction threshold,
+the kubelet reclaims the resource until the signal reaches the threshold of `1Gi`,
+and then continues to reclaim the minimum amount of `500Mi` it until the signal
+reaches `1.5Gi`. 
+
+Similarly, the kubelet reclaims the `imagefs` resource until the `imagefs.available`
+signal reaches `102Gi`. 
+
+The default `eviction-minimum-reclaim` is `0` for all resources.
+
+## Scheduler
+
+The scheduler respects reported node conditions when making scheduling decisions.
+
+For example, if `MemoryPressure` node condition is active on a node, the scheduler
+does not schedule new `BestEffort` pods on the node. If the `DiskPressure` node
+condition is active, the scheduler does not schedule any new pods on the node. 
```

Comment:
This section is not really about eviction.

Reply:
Maybe move it to https://kubernetes.io/docs/concepts/architecture/nodes/ or check that the content there already covers this?